src/components/Shared/UserChips.jsx

```diff
@@ -58,22 +58,16 @@ export function SiteAdminChip({ person }) {
 export function BannedUserChip({ person }) {
   if (!person.banned) return null;
 
-  return <SquareChip color={"danger"} tooltip="User is banned" iconOnly={<BlockIcon fontSize="small" />} />;
+  return <SquareChip color={"danger"} tooltip="Banned" iconOnly={<BlockIcon fontSize="small" />} />;
 }
 
 export function BotAccountChip({ person }) {
   if (!person.bot_account) return null;
 
-  return (
-    <SquareChip
-      color={"warning"}
-      tooltip="User is bot account"
-      iconOnly={<SmartToyIcon fontSize="small" />}
-    />
-  );
+  return <SquareChip color={"warning"} tooltip="Bot Account" iconOnly={<SmartToyIcon fontSize="small" />} />;
 }
 export function DeletedUserChip({ person }) {
   if (!person.deleted) return null;
 
-  return <SquareChip color={"danger"} tooltip="User is deleted" iconOnly={<DeleteIcon fontSize="small" />} />;
+  return <SquareChip color={"danger"} tooltip="Beleted User" iconOnly={<DeleteIcon fontSize="small" />} />;
```

Comment:
Is it wanted to be "Beleted User?"

Reply:
thanks :)

Reply:
I'm also interested in maybe looking at I18N for this too - we'd have to pull ALL the strings out :/